action-src/main.ts

```diff
@@ -107,7 +107,7 @@ async function runChromatic(options): Promise<Output> {
     url: ctx.build?.webUrl,
     code: ctx.exitCode,
     buildUrl: ctx.build?.webUrl,
-    storybookUrl: ctx.build?.cachedUrl,
+    storybookUrl: ctx.build?.cachedUrl.replace(/iframe\.html.*$/, ''),
```

Comment:
Why the `.*` at the end?

PS we also have a `build.storybookUrl` that we could request, perhaps could be more reliable?

Reply:
I added that to also strip off any query params (which would not work for the manager)

Reply:
There should never be any query params on cachedUrl I think, but can't hurt to guard against it I suppose.